Sources/LiveKit/Track/AudioManager.swift

```diff
@@ -121,6 +123,12 @@ public class AudioManager: Loggable {
         set { _state.mutate { $0.isSpeakerOutputPreferred = newValue } }
     }
 
+    /// If this is set, this will be used instead of dynamic configuration.
+    public var sessionConfiguration: AudioSessionConfiguration? {
```

Comment:
you should add a comment to the customConfigureFunc as well noting that if you set it then the sessionConfiguration and isSpeakerOutputPreferred properties will be ignored